src/platform/functions/index.js

```diff
@@ -6,7 +6,7 @@ admin.initializeApp();
 
 const timeout = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
-exports.scrapeEventbrite = functions
+exports.scrapeEventGiveCity = functions
```

Comment:
hmm, the changes in this file is from a accidental commit from Goutam, can you merge master again?